src/doc/trpl/traits.md

```diff
@@ -19,7 +19,8 @@ impl Circle {
 
 [methodsyntax]: method-syntax.html
 
-Traits are similar, except that we define a trait with just the method
+[Traits](../reference.html#traits) are similar,
```

Comment:
I mean where in the chapter should it be defined? Maybe before the first paragraph?